packages/dapp/components/utils/contracts.ts

```diff
@@ -140,3 +145,23 @@ export const getUbiquityFormulasContract = (address: string, provider: ethers.pr
 export const getUbiquityGovernanceContract = (address: string, provider: ethers.providers.Provider) => {
   return getContract(UbiquityGovernance.abi, address, provider);
 };
+
+export const getuCRTokenContract = (address: string, provider: ethers.providers.Provider) => {
+  return getContract(uCRToken.abi, address, provider);
+};
+
+export const getuADTokenContract = (address: string, provider: ethers.providers.Provider) => {
+  return getContract(uADToken.abi, address, provider);
+};
```

Comment:
I don't see the full file conveniently in this view on GitHub, but I believe that we already have redundant methods (for example, for uCR it is mistakenly called getUbiquityAutoRedeem)